### DISCUSSION TITLE
"mail merge" new pages

### DISCUSSION THREAD

**Discussion by jmls** [dolanmiu/docx#2529] [General]
Would I be able to use this library to read an existing docx, copy the pages into a string (document.xml) , then delete those pages. I want to then run through a json data set, using mustache or gomplate or equivalent and "merge" the json data into the string. Once the merge is complete, I want to then take that string and use it to crteate a new page and into it the docx

So eventually I would end up with a docx of say 10 pages, one page for each item in the top level array

**Comment by dolanmiu** [comment]
> copy the pages into a string
Not possible nope

This library only allows the API -> word document

Reading and editing is limited (there is patchDocument though which is good, but may not be what you're looking for
